internal/pkg/template/templates/addons/aurora/rdws/cf.yml

```diff
@@ -0,0 +1,173 @@
+Parameters:
```

Comment:
View diff for the file here: https://gist.github.com/efekarakus/17f6a13faa8ee4670239844b2b1cc047